google/cloud/security/inventory/pipelines/load_projects_iam_policies_pipeline.py

```diff
@@ -53,21 +56,24 @@ def run(dao=None, cycle_timestamp=None, configs=None, crm_rate_limiter=None):
         raise LoadDataPipelineError(e)
 
     crm_client = CloudResourceManagerClient(rate_limiter=crm_rate_limiter)
-    try:
-        # Retrieve data from GCP.
-        # Flatten the iterator since we will use it twice, and it is faster
-        # than cloning to 2 iterators.
-        iam_policies_map = crm_client.get_project_iam_policies(
-            RESOURCE_NAME, project_numbers)
-        # TODO: Investigate improving so the pylint disable isn't needed.
-        # pylint: disable=redefined-variable-type
-        iam_policies_map = list(iam_policies_map)
-
-        # Flatten and relationalize data for upload to cloud sql.
-        flattened_iam_policies = (
-            transform_util.flatten_iam_policies(iam_policies_map))
-    except ApiExecutionError as e:
-        raise LoadDataPipelineError(e)
+
+    # Retrieve data from GCP.
+    # Not using iterator since we will use the iam_policy_maps twice.
+    iam_policy_maps = []
+    for project_number in project_numbers:
+        try:
+            iam_policy = crm_client.get_project_iam_policies(
+                RESOURCE_NAME, project_number)
+            iam_policy_map = {'project_number': project_number,
```

Comment:
This looks like it would be better as a namedtuple, then the keys/attributes are explicitly defined ahead of time, which is cleaner than a dict with potentially variable keys, since any new keys have to be defined in the namedtuple definition before use: https://docs.python.org/2/library/collections.html#collections.namedtuple. Can be done in a future CL.